napari/layers/base/base.py

```diff
@@ -29,6 +29,16 @@
 Extent = namedtuple('Extent', 'data world step')
 
 
+def no_op(layer, even):
+    """
+    A convenient no-op event for the layer mouse binding.
+
+    This makes it easier to handle many cases by inserting this as
+    as place holder
+    """
```

Comment:
Can you add `Parameters` / `Returns` here

Reply:
Done.